1.9.0/values.yaml

```diff
@@ -0,0 +1,159 @@
+# Default values for DirectorOnPrem.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Update platfrom according to your Platform, Default include(GKE,Packet) and SUSE
+platform: Default
+
+server:
+  release: 1.9.0-ee
+  dockerSecret: directoronprem-registry-secret
+  protocol: http
+  url:
+  httpNodePort: 30380
```

Comment:
If we installed ingress as ds then is these port required?

Reply:
This needs to be fixed, we are aware. Will take up as part of 1.10. There are some other protocol and url related streamlining that needs to happen.

Reply:
We will keep it for this release since it will be required if ingress is deployed as deployment and won't affect anything if its ds. cc @atulabhi